Formula/pktanon.rb

```diff
@@ -1,7 +1,7 @@
 class Pktanon < Formula
   desc "Packet trace anonymization"
-  homepage "https://www.tm.uka.de/software/pktanon/index.html"
-  url "https://www.tm.uka.de/software/pktanon/download/pktanon-1.4.0-dev.tar.gz"
+  homepage "http://www.tm.kit.edu/software/pktanon/index.html"
+  url "http://www.tm.kit.edu/software/pktanon/download/pktanon-1.4.0-dev.tar.gz"
```

Comment:
no HTTPS?

Reply:
yeah, no https.

Reply:
I just checked and HTTPS works for me on both links.

Reply:
oh, interesting, it works for me now as well. 🤦

Reply:
I should attach the curl test when i filed the PR 😄 
```
$ curl -I https://www.tm.kit.edu/software/pktanon/index.html
HTTP/1.1 200 OK
Date: Mon, 26 Jun 2023 14:48:45 GMT
Server: Apache
Accept-Ranges: bytes
Vary: Accept-Encoding
Content-Length: 5536
Content-Type: text/html
```